nextflow_schema.json

```
@@ -566,6 +566,11 @@
                     "default": true,
                     "description": "Reduce GTDB-Tk memory consumption by running pplacer in a setting writing to disk.",
                     "help_text": "Will be slower. Set to `false` to turn this off."
+                },
+                "genomad_db": {
+                    "type": "string",
+                    "default": "https://portal.nersc.gov/genomad/__data__/genomad_db_v1.1.tar.gz",
+                    "description": "Database for virus classification with geNomad"
```

It should actually be a folder! So it should look something like this?
```suggestion
                "genomad_db": {
                    "type": "string",
                    "default": "https://portal.nersc.gov/genomad/__data__/genomad_db_v1.1.tar.gz",
                    "description": "Database for virus classification with geNomad",
                    "help_text": "Must be a directory containing the uncompressed contents from https://portal.nersc.gov/genomad/__data__/genomad_db_v1.1.tar.gz"
```